### DISCUSSION TITLE
Lost the option to donate just to OsmAnd?

### DISCUSSION THREAD

**Discussion by mattmccutchen** [osmandapp/OsmAnd#13525] [General]
I'm interested in donating to OsmAnd.  Buying things on the Google Play Store is not ideal because of privacy and the percentage that Google takes, and if I have a specific amount I want to donate, it's clunky to try to select a combination of purchases that adds up to that amount.  In the past, I donated via Bitcoin; IIRC, https://osmand.net/osm_live had two Bitcoin addresses, one for a donation just to OsmAnd and one for a donation split between OsmAnd and OSM contributors.  Now https://osmand.net/help-online/faq#buy_app still links to https://osmand.net/osm_live, but that page redirects to https://osmbtc.org/, and that page makes no mention of any of the money going to OsmAnd; it sounds like the money may go only to OSM contributors.

Am I missing something?  If not, would the project be interested in offering a direct donation method (Bitcoin or something else) for OsmAnd only again?

(I'm aware that a significant percentage of Bitcoin donations can be lost to various processing fees, especially for small donations, and this might even be worse than the percentage taken by the Google Play Store.  Still, a Bitcoin option could be useful for large donations, and maybe people will find ways to reduce or avoid the Bitcoin fees.)

**Update (2025-08-16)**: Looks like #14672 is probably the best place to upvote now.

**Comment by vshcherb** [comment]
Donation to OsmAnd is left only on F-Droid page. It's another BTC address for OsmAnd Team - 1GRgEnKujorJJ9VBa76g8cp3sfoWtQqSs4 (https://github.com/osmandapp/osmandapp.github.io/blob/fb3c5aa4a6ee9cb4c6d1dc0d4bf2c55d4bade5d4/website/templates/pub/osm_live.html#L231)

**Reply by mattmccutchen** [reply]
Thanks for providing the address.  You said it's "on F-Droid page": what URL do you mean?  I don't see it on https://f-droid.org/en/packages/net.osmand.plus/.  It looks like the file you linked is just a "template" that may no longer be used by the https://osmand.net web site now that the https://osmand.net/osm_live page has been deleted and replaced with a redirect to https://osmbtc.org.  It would be good to have the address in an official place on the OsmAnd web site, not just in a discussion thread or some random file in a repository that users may not be confident is still valid and will not lose their money.

I might be able to help with this by submitting a PR if you give me some guidance about what you would accept.  Maybe the address should just be added as another item in the list at https://osmand.net/help-online#buy_app (since the GitHub and F-Droid sponsorship links both point there), and the existing link to https://osmbtc.org titled "Bitcoin" should be clarified to state whether it goes to OsmAnd, map contributors, or some split.

**Reply by Bibi56** [reply]
@mattmccutchen you say:  I don't see it on https://f-droid.org/en/packages/net.osmand.plus/. 
But I see a section "Donate" after the section "Anti-Features". Did you stop your reading after the horizontal line?
After that I see a link to a bitcoin account.
https://osmand.net/help-online/#buy_app gives me a 403. Time to close this discussion?

**Reply by mattmccutchen** [reply]
Hm, I don't know if I missed the Bitcoin address on the F-Droid page or it somehow didn't show up for me.  Anyway, my argument stands that if OsmAnd still considers Bitcoin an officially supported donation method, it should be documented officially on their own web site, not just a third party site.  I wouldn't advise people to donate to cryptocurrency addresses that appear only on third party sites; in my opinion, it doesn't provide enough confidence that the money will still go to the right place.

[As discussed elsewhere](https://github.com/osmandapp/OsmAnd/issues/17681#issuecomment-3012141854), OsmAnd has a new purchase option based on FastSpring.  I made [a merge request to add it to F-Droid](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/26358).  I'm also proposing to remove the Bitcoin address from F-Droid now that we have a better option.  @vshcherb Are you OK with these changes to F-Droid? If I don't hear from the OsmAnd team in a few weeks, I'll unblock the F-Droid merge request.  Thanks.

_Update (2025-08-27):_ I started a separate discussion thread (#23354) rather than just relying on a ping to vshcherb.

**Reply by EugeneZmeuk** [reply]
I'm from OsmAnd management. 
It makes sense to change the donation method to www.osmand.net/pricing
